app/Http/Controllers/Users/UsersController.php

```diff
@@ -93,8 +93,8 @@ public function store(SaveUserRequest $request)
         $this->authorize('create', User::class);
         $user = new User;
         //Username, email, and password need to be handled specially because the need to respect config values on an edit.
-        $user->email = e($request->input('email'));
-        $user->username = e($request->input('username'));
+        $user->email = trim($request->input('email'));
```

Comment:
I think it’s possible to have an email address with an apostrophe in the ‘username’ portion - `snipey’head@domain.com` - so should we maybe keep the `e()` in there as well, just in case? E.g. `e(trim($request->input(‘email’))`

Reply:
Or are we over-escaping? we’re setting an attribute on a User object. That can have as many funky characters as you want, just so long as you escape back on the way out, right?

Reply:
I think that would be over-escaping, which is why I removed it. Also I think Laravel's email email validation rejects emails with apostrophe anyway, so as it stands, those emails would't validate anyway, but even if they did, we'd want to escape it only on the way out if we're printing it somewhere.